source/Background/Keyring.js

```diff
@@ -122,6 +122,7 @@ export const getKeyringHandler = (type, keyring) => ({
   },
   [HANDLER_TYPES.GET_ASSETS]: async ({ refresh }) => {
     try {
+      if (!keyring?.isUnlocked) return;
```

Comment:
what happens after this? shouldnt it redirect to login screen ?

Reply:
Not really, the issue happens most likely because a miss-fire on the action or some kind of race condition between switching the states, it shouldn't affect normal functionality but it will prevent the error from showing up when it shouldn't